Gemfile

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
We don't want this comment. You can disable rubocop rules.
Add this to the **rubocop.yml**
```
Style/FrozenStringLiteralComment:
  Enabled: false
```

Reply:
Also remove all the comments from all files. Sorry for that :grimacing: